server/src/main/scala/za/co/absa/atum/server/api/http/Routes.scala

```diff
@@ -111,6 +111,26 @@ trait Routes extends Endpoints with ServerOptions {
           PartitioningController.getFlowPartitionings(flowId, limit, offset)
         }
       ),
+      createServerEndpoint[
+        (Long, Option[Int], Option[Long]),
+        ErrorResponse,
+        PaginatedResponse[PartitioningWithIdDTO]
+      ](
+        getAncestorsEndpointV2,
+        { case (partitioningId: Long, limit: Option[Int], offset: Option[Long]) =>
+          PartitioningController.getAncestors(partitioningId, limit, offset)
+        }
+      ),
+//      createServerEndpoint[
+//        (Long, Long, String),
+//        ErrorResponse,
+//        SingleSuccessResponse[ParentPatchV2DTO]
+//      ](
+//        patchPartitioningParentEndpointV2,
```

Comment:
I think that getAncestors server scala part is implemented here in this PR, but that was implemented in https://github.com/AbsaOSS/atum-service/pull/312. And what was supposed to be implemented here, i.e. the Patch endpoint, is commented out.

Please prepare the PR for the review, I'm more than happy to review afterwards